commands.js

```diff
@@ -410,6 +410,22 @@ let captureDom = (win) => {
         cy.writeFile(`./${vtConfFile.debug}/${imageName}-${imageType}/${imageName}.json`, dom);
     }
 };
+let scrollToTop = (win) =>{
+    let tries = 0;
+    let scrollOffset = 1;
+    do {
+        // Translate to the top of the page and then capture the dom
+        win.eval(`document.body.style.transform="translateY(0)"`);
+        cy.wait(250);
+        scrollOffset = win.eval(`scrollOffset = document.body.style.transform`);
+        tries += 1;
+    }while (scrollOffset = 0 && tries < 40)
+    if (tries < 40){
+        cy.task('logger', {type: 'info', message: `Scroll offset is 0 after ${tries} tries`});
+    }else{
+        throw new Error(`Couldn't scroll to the top of page`);
+    }
+}
```

Comment:
It would be nice to show that this function works too at the end and add the proof to jira card

Reply:
Recreating this issue is very hard to replicate and random it sometimes occurring when doing a lot of tests. So I don't think I will be able to reliably check that.